charts/puris/charts/backend/templates/deployment.yaml

```diff
@@ -62,12 +64,24 @@ spec:
               value: "{{ .Values.puris.edc.idsurl }}"
             - name: MINIKUBE_IP
               value: "{{ .Values.puris.minikube.ip }}"
+            - name: OWN_BPNA
+              value: "{{ .Values.puris.own.default.bpna }}"
```

Comment:
Some of the own.default fields do have it but others don't. Could you please 
a) create an issue for harmonization
b) directly remove the default (all application properties with refactoring in intelliJ to also update the classes (VariablesServices, etc.), docker compose files and helm related files).

Reply:
Maybe it makes sense to also update some of the default values in helm -> e.g. the edc address to something like http://YOUR-EDC-MANAGEMENT-URL/